addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -31,6 +31,7 @@
                             canChangeDefinition="state.canChangeDefinition"
                             type="propertyConfiguration.type"
                             string="propertyConfiguration.string"
+                            showAvatar="propertyConfiguration.show_avatar"
```

Comment:
Linting: try to keep alphabetical ordering when possible :) just for convention ^^